Go/Go.sublime-settings

```diff
@@ -0,0 +1,11 @@
+{
+  // Prefer tabs because gofmt enforces tabs.
+  "translate_tabs_to_spaces": false,
+  // Trigger autocompletion on dot access.
+  "auto_complete_triggers": [
```

Comment:
Many packages use dots for attribute access, however none of them defines `auto_complete_triggers`. I'm not sure whether they should or should not, but this is something we should discuss, imo.

Reply:
This certainly could be removed. I'll run without this setting for a few days to get a feel.

Reply:
Removed the trigger setting for now.